src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.sps.data.Meal;
+
+@WebServlet("/meal/*")
+public class MealServlet extends HttpServlet {
+
+    private ArrayList<Meal> dishes;
+
+    @Override
+    public void init() {
+        dishes = new ArrayList<>(Arrays.asList());
+        dishes.add(new Meal(0, "fried potato", "fried potato with mushrooms and onion",
+            new ArrayList<>(Arrays.asList("potato", "onion", "mushrooms", "oil")), "Main"));
+        dishes.add(new Meal(1, "Italian pizza", "Pizza with pineaple, sausage and tomato",
+            new ArrayList<>(Arrays.asList("flour", "water", "sausage", "tomato", "pineaple", "cheese")), "pizza"));
+        dishes.add(new Meal(2, "vegetable soup", "vegetable soup with onion",
+            new ArrayList<>(Arrays.asList("potato", "onion", "cabbage", "mushrooms", "water", "carrot", "pumpkin")), "soup"));
+        dishes.add(new Meal(3, "chocolate cake", "chocolate cake with butter cream and strawberry",
+            new ArrayList<>(Arrays.asList("flour", "water", "butter", "cacao", "chocolate", "sugar", "strawberry", "eggs")), "dessert"));
+        dishes.add(new Meal(4, "hot chocolate", "hot chocolate with sugar and caramel",
+            new ArrayList<>(Arrays.asList("cacao", "sugar", "milk", "caramel", "vanil")), "drinks"));
+    }
+
+    /*@Override
```

Comment:
Let's aim to remove dead code, rather than commenting it.

Reply:
Do you think is it better to remove few commits where I started develop REST API and merge servlet that I had before, like Olga did? And start develop REST in new PR, so we could change same file.

Reply:
Well, removing commits is somewhat tedious, and we're already in the middle of a review, so I'd probably suggest continuing with what we already have.

Reply:
... having said that, it's up to you. Choose whatever's more convenient.